robot_framework/linear_framework.py

```diff
@@ -22,11 +22,12 @@ def main():
     orchestrator_connection.log_trace("Robot Framework started.")
     initialize.initialize(orchestrator_connection)
 
+    queue_element_count = [0]  # A count of queue elements to keep the robot from running for too long, in a tuple to make it mutable
```

Comment:
```suggestion
    queue_element_count = [0]  # A count of queue elements to keep the robot from running for too long, in a list to make it mutable.
```

tuples are not mutable ;)